upgrading/manual_upgrades.adoc

```diff
@@ -507,80 +507,41 @@ updated.
 [[updating-policy-definitions]]
 == Updating Policy Definitions
 
-After a cluster upgrade, the default roles
+During a cluster upgrade, the default roles
 xref:../architecture/additional_concepts/authorization.adoc#roles[default
-cluster roles] are automatically updated. To check if all defaults are set as
-recommended for your environment, run:
-
-----
-# oc adm policy reconcile-cluster-roles
-----
+cluster roles] are automatically updated. 
 
 [WARNING]
 ====
-If you have customized default cluster roles and want to ensure a role reconciliation
-does not modify those customized roles, annotate them with `openshift.io/reconcile-protect`
-set to `true` when using the old Openshift policy format. When using the new RBAC
-roles, use `rbac.authorization.kubernetes.io/autoupdate` set to `false` instead.
-In doing so, you are responsible for manually updating those roles with any new
-or required permissions during upgrades.
-====
-
-This command outputs a list of roles that are out of date and their new proposed
-values. For example:
-
-----
-# oc adm policy reconcile-cluster-roles
-apiVersion: v1
-items:
-- apiVersion: v1
-  kind: ClusterRole
-  metadata:
-    creationTimestamp: null
-    name: admin
-  rules:
-  - attributeRestrictions: null
-    resources:
-    - builds/custom
-...
-----
+If you customized default cluster roles and want to ensure a role reconciliation
+does not modify those customized roles, set the
+`rbac.authorization.kubernetes.io/autoupdate` parameter in the 
+to `false`.
 
-[NOTE]
-====
-Your output will vary based on the OpenShift version and any local
-customizations you have made. Review the proposed policy carefully.
+You must manually update the roles with this setting to include any new
+or required permissions after upgrading.
 ====
 
-You can either modify this output to re-apply any local policy changes you have
-made, or you can automatically apply the new policy using the following process:
 
-. Reconcile the cluster roles:
-+
-----
-# oc adm policy reconcile-cluster-roles \
-    --additive-only=true \
-    --confirm
-----
+To check if all defaults are set as recommended for your environment, run the
```

Comment:
Until the reconcile command has a proper --dry-run setting in a later release, what follows is not a "check" but will actually reset the cluster roles and rolebindings to their default values. Custom changes to the cluster roles would need to be applied to the policy file before running reconcile (or changed manually afterwards).